Makefile

```diff
@@ -7,10 +7,10 @@ AZURE_IMAGE ?= quay.io/openshift-on-azure/azure:$(GITCOMMIT)
 
 all: azure
 
+# you must login to the ci cluster before running the secrets target
```

Comment:
Is there a way that we can add a check to verify that the user is authenticated here rather than adding this as a comment?

Reply:
It would even be nicer if the user actually got logged in automatically using a combination of secrets stored in the local `secrets` directory. 
For what its worth, we can use `oc whoami` to check for the currently logged in user or we might want to parse `$HOME/.kube/config` to see if the user is logged in but then we have to deal with possible invalidated sessions/tokens and it all ends up being too much work. 

I am open to taking out this comment if it makes you and possibly others happier with this change.